src/components/Navbar/DonateBtn.js

```diff
@@ -92,13 +80,25 @@ class DonateBtn extends React.Component {
         onMouseOver={this.handleMouseOver}
         onMouseOut={this.handleMouseOut}
         onClick={this.handleOnClick}
+        style={{
+          height: this.props.buttonHeight + "px", width: this.props.buttonWidth + "px", minWidth: "96px"
+        }}
       >
-        <StyledText>Donate</StyledText>
-        <StyledImg
-          src={this.state.imgSrc}
-          alt="heart color"
-          onClick={this.handleOnClick}
-        />
+        <span
+          style={{width:"fit-content", margin:"auto"}}
+        >
+          <StyledFont
+            style={{ fontSize: this.props.buttonFontSize + "px" }}
+          >
+            Donate
+            </StyledFont>
+          <StyledImg
+            src={this.state.imgSrc}
+            alt="heart color"
```

Comment:
i think this goes to @rishFilet  (no need to cater follg comment within this PR)
the alt text "heart color" - is this from figma? do we need a separate to-do task to work out the accessibility parameters of the website?